misc/task_handler.c

```diff
@@ -20,7 +20,7 @@
 #include "thermalmgmt.h"
 #endif
 
-LOG_MODULE_DECLARE(pwrmgmt, CONFIG_PWRMGT_LOG_LEVEL);
+LOG_MODULE_DECLARE(pwrmgmt, CONFIG_EC_LOG_LEVEL);
```

Comment:
better to call this as thrdmgmt. i.e. LOG_MODULE_DECLARE(thrdmgmt, CONFIG_EC_LOG_LEVEL);

Reply:
I get the point! thanks you.